tests/test.py

```diff
@@ -0,0 +1,31 @@
+import pybamm
+import pybamm_tea as TEA
```

Comment:
not good naming to import the package and give it the same name as a class it defines. did you mean to do `import TEA from pybamm_tea`?

Reply:
see above comment. did you check these tests run and pass?